### PR TITLE
Changes archives examine message

### LINKED ISSUE #36550: Curators can speak literally any language, but the Archives show up as gibberish.
This isn't a one-time error; it's more like an oversight. If anyone who isn't a cultist tries to use the Archives, they get the message, "All of these books seem to be gibberish." However, the person most likely to click on the Archives is the Curator, who knows literally every language, **including gibberish**.
I'm in no way suggesting that the Curator should be able to use the Archives (that would be stupid and overpowered), but it seems stupid for it to show up to them as unreadable.

### PR DESCRIPTION
**Pull request by Cyberboss**
Fixes #36550

### CODE REVIEW

**Comment by Buggy123** [comment]
This doesn't solve the fact that the curator, who (to my understanding) is supposed to speak all languages, doesn't know it.

File: code/modules/antagonists/cult/cult_structures.dm [hunk 1]
```diff
@@ -235,7 +235,7 @@
 
 /obj/structure/destructible/cult/tome/attack_hand(mob/living/user)
 	if(!iscultist(user))
-		to_chat(user, "<span class='warning'>All of these books seem to be gibberish.</span>")
+		to_chat(user, "<span class='warning'>These books are written in a language you cannot understand.</span>")
```
Comment (Jalleo) [inline code/modules/antagonists/cult/cult_structures.dm]: How about "These books wont open and it hurts to even try and read the covers."

**Comment by Buggy123** [comment]
That works.

**Comment by arsserpentarium** [comment]
Curator-->warlock would be better.

**Pull request merged**